coalib/parsing/DefaultArgParser.py

```diff
@@ -187,6 +188,12 @@ def default_arg_parser(formatter_class=None):
         help='filters `--show-bears` by the given languages')
 
     outputs_group.add_argument(
+        '--filter-by', action='append', nargs='+',
+        help='filters `--show-bears` by the filter given as argument.'
+             'Available filters: {}'.format(
```

Comment:
missing space in string after `argument.` / before `Available` :)
Renders as
```
... by the filter given as argument.Available filter: ...
```
currently